Microsoft.Azure.Devices.Edge.sln

```diff
@@ -189,7 +189,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edg
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edge.Test", "test\Microsoft.Azure.Devices.Edge.Test\Microsoft.Azure.Devices.Edge.Test.csproj", "{08986FED-9283-47F1-B938-86D9C6E753E1}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest", "edge-agent\test\Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest\Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.csproj", "{79540E35-45EA-4568-9AB7-C1CC0BFF73EE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent", "edge-agent\src\Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent\Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.csproj", "{47AA5965-5826-489D-B74B-314A61629121}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.Test", "edge-agent\test\Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.Test\Microsoft.Azure.Devices.Edge.Agent.DiagnosticsComponent.Test.csproj", "{B736B03C-1185-4AE5-87B7-665C6790F50F}"
```

Comment:
Can you set this up on the device at your desk (or any small device like a Pi) and let it run over the weekend with some sort of additional workload on it? I am really worried about longterm perf on smaller devices - Ideally, this would be done before this is checked in.

Reply:
yea, I'll set it up to run over the weekend. I'm not too worried though, in the normal configuration the scraping only runs once an hour and the upload once a day.